seqfold/dna.go

```diff
@@ -0,0 +1,519 @@
+package seqfold
+
+var DNA_COMPLEMENT = Comp{'A': 'T', 'T': 'A', 'G': 'C', 'C': 'G', 'N': 'N'}
+
+// a, b, c, d in a linear multi-branch energy change function.
+// Inferred from:
+// Supplemental Material: Annu.Rev.Biophs.Biomol.Struct.33:415-40
+// doi: 10.1146/annurev.biophys.32.110601.141800
+// The Thermodynamics of DNA Structural Motifs
+// SantaLucia and Hicks, 2004
+var DNA_MULTIBRANCH = MultiBranchEnergies{A: 2.6, B: 0.2, C: 0.2, D: 2.0}
+
+// The Thermodynamics of DNA Structural Motifs
+// SantaLucia and Hicks, 2004
+var DNA_NN = BpEnergy{
```

Comment:
what does NN stand for?

Reply:
I should put together an explainer for the energy parameters.

Reply:
I have no idea what `NN` stands for 😅